cron/hourly/70-handle-batch-subscriptions-update.ts

```diff
@@ -102,13 +122,19 @@ export async function run() {
               subscription: order.Subscription,
               collective: order.collective.minimal,
               fromCollective: order.fromCollective.minimal,
-              reasonCode: reasonCode,
-              reason: reason,
+              reasonCode: reason.code,
+              reason: reason.message,
+              messageForContributors: order.data?.messageForContributors,
               order: order.info,
               tier: order.Tier?.info,
+              awaitForDispatch: true, // To make sure we won't kill the process while emails are still being sent
```

Comment:
Added this flag to make sure we await the email dispatch in `Activities.afterCreate`, without it the script could exit before all the emails have been sent.